doc/source/cd/methods/onlinefetdrift.ipynb

```diff
@@ -0,0 +1,175 @@
+{
```

Comment:
<p>"starting from&nbsp;<code>t=min(windows_sizes)-1</code>" -&gt; "starting from&nbsp;<code>t=min(windows_sizes)</code>".   I think this is less confusing and the <code>-1</code>  can just be incorporated into the definition of runtime.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>